uber/jinja.py

```diff
@@ -0,0 +1,61 @@
+from uber.common import *
+
+
+class JinjaEnv:
+    _env = None
+    _exportable_functions = {}
+    _filter_functions = {}
+    _template_dirs = []
+
+    @staticmethod
+    def append_template_dir(dirname):
+        """
+        Add another template directory we should search when looking up templates.
+        """
+        JinjaEnv._template_dirs.insert(0, dirname)
+
+    @staticmethod
+    def env():
```

Comment:
You can say

```python
from sideboard.lib import on_startup

@on_startup
def some_initialization_stuff():
    ...
```

That will cause the code to run when CherryPy starts, after everything has been imported.  We already do this in a few places, though most of the time we just do stuff at the module-level which happens at import time.